src/api-client/mock/data/collection.json

```diff
@@ -82,13 +85,120 @@
       "type": "Delete",
       "actor": {
         "type": "Person",
-        "name": "Sally"
+        "name": "Sally",
+        "id": "http://sally.example.org"
       },
       "object": "http://example.org/notes/1",
       "origin": {
         "type": "Collection",
         "name": "Sally's Notes"
       }
+    },
+    {
+      "type": "Article",
+      "name": "Might the force be with you",
+      "content": "\"May the Force be with you\" was a phrase used to wish an individual or group good luck or good will, one that expressed the speaker's wish that the Force work in the favor of the addressee. The phrase was often used as individuals parted ways or in the face of an impending challenge.",
+      "attributedTo": "https://mastodon.social/users/fraenki"
+    },
+    {
+      "type": "Article",
+      "name": "Memory, time, and technology",
+      "content": "Cole has been thrust from another world into ours and he's confronted by the confusion we live in, which most people somehow accept as normal. So he appears abnormal, and what's happening around him seems random and weird. Is he mad or are we?",
+      "attributedTo": "https://mitra-social.herokuapp.com/user/franco.g"
+    },
```

Comment:
Think this is never going to happen. We will always have an activity (Create, Update, Delete, Follow, etc.) in this collection. We should ensure this in backend if we get data pushed in into an inbox.

Reply:
Should I delete that now? Because there is still the possibility with this variant. Unless I change that in the store too

Reply:
The backend rejects content with types that are not activities. This will never happen.